spec.html

```diff
@@ -5219,6 +5219,17 @@
             </tr>
             <tr>
               <td>
+                [[GlobalThisValue]]
+              </td>
+              <td>
+                Object
+              </td>
+              <td>
+                The value returned by `this` in global scope. This may be the same as the global object, or may not. For example, in HTML the `this` value is a `WindowProxy` object, whereas the global object is a `Window`.
```

Comment:
On reflection I agree that the "usually" clause is bad. Just remove it. It adds as much information as the "It may... or may not" clause :-P